app/database/models.py

```diff
@@ -211,66 +211,93 @@ class SalarySettings(Base):
     # Code revision required after categories feature is added
     # Code revision required after holiday times feature is added
     # Code revision required after Shabbat times feature is added
-    __tablename__ = "salary_settings"
+    __tablename__ = 'salary_settings'
 
     user_id = Column(
-        Integer, ForeignKey("users.id"), primary_key=True,
+        Integer,
+        ForeignKey('users.id'),
+        primary_key=True,
     )
     # category_id = Column(
     #     Integer, ForeignKey("categories.id"), primary_key=True,
     # )
     category_id = Column(
-        Integer, primary_key=True,
+        Integer,
+        primary_key=True,
     )
     wage = Column(
-        Float, nullable=False, default=SalaryConfig.MINIMUM_WAGE,
+        Float,
+        nullable=False,
+        default=SalaryConfig.MINIMUM_WAGE,
     )
     off_day = Column(
-        Integer, CheckConstraint("0<=off_day<=6"), nullable=False,
+        Integer,
+        CheckConstraint('0<=off_day<=6'),
+        nullable=False,
         default=SalaryConfig.SATURDAY,
     )
     # holiday_category_id = Column(
```

Comment:
can remove?

Reply:
This is not part of my feature actually. This is in the develop. The changes are only because of the new autoformatter Yam put in the prehooks

Reply:
Though I don't mind removing it but I don't know who has been using this :)